FilteringStoryboard/FilteringStoryboard/FilterPhotoViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  FilterPhotoViewController.swift
+//  FilteringStoryboard
+//
+//  Created by Ezra Black on 5/5/20.
+//  Copyright © 2020 Casanova Studios. All rights reserved.
+//
+import CoreImage
+import UIKit
+
+class ViewController: UIViewController, UIImagePickerControllerDelegate, UINavigationControllerDelegate {
+    @IBOutlet var imageView: UIImageView!
+    @IBOutlet var intensity: UISlider!
+    var currentImage: UIImage!
+    
+    var currentFilter: CIFilter!
```

Comment:
Since you set this in `viewDidLoad`, there's not much of a reason to make this implicitly unwrapped; you could just move that initialization to the declaration. (`var currentFilter = CIFilter(name: "CISepiaTone")`) Much safer that way.